packages/edit-site/src/components/header/document-actions/index.js

```diff
@@ -71,11 +79,19 @@ function useSecondaryText() {
 	return {};
 }
 
-export default function DocumentActions( { documentTitle } ) {
+export default function DocumentActions( { template } ) {
+	const documentTitle = template?.slug;
```

Comment:
Same here, we could use the title logic from the navigation panel here too. See my other comment below. Not a blocker, just a thought 😄